qutebrowser/browser/qutescheme.py

```diff
@@ -135,6 +135,11 @@ def data_for_url(url):
     Return:
         A (mimetype, data) tuple.
     """
+    # normalize path and strip redundant trailing slashes
+    norm_url = url.adjusted(QUrl.NormalizePathSegments | QUrl.StripTrailingSlash)
+    if norm_url != url:
+       raise Redirect(norm_url)
```

Comment:
This should be indented by one space more (four, not three spaces).

Reply:
Argh, how did I miss this? My bad, sorry.